pkg/virt-handler/device-manager/device_controller_test.go

```diff
@@ -211,5 +217,26 @@ var _ = Describe("Device Controller", func() {
 				return exists1 || exists2
 			}, 5*time.Second).Should(BeFalse())
 		})
+
+		It("Should not remove permanent device plugins if permittedHostDevices is removed from the CR", func() {
+			emptyConfigMap, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(&v1.KubeVirtConfiguration{})
+			Expect(emptyConfigMap.GetPermittedHostDevices()).To(BeNil())
+
+			initialDevices := []Device{plugin1, plugin2}
+			deviceController := NewDeviceController(host, maxDevices, permissions, initialDevices, emptyConfigMap, clientTest.CoreV1())
+
+			go deviceController.Run(stop)
+
+			// Ensure that the controller has time to start first
+			time.Sleep(1 * time.Second)
```

Comment:
This is redundant.

Reply:
Looking over the code again you are right, since i changed the test condition.